### PR TITLE
[YAML] Fixed script issues

### PR DESCRIPTION
**Pull request by manjunath-grl**
`Fixes`:  [309](https://github.com/project-chip/matter-test-scripts/issues/309), [316](https://github.com/project-chip/matter-test-scripts/issues/316), [314](https://github.com/project-chip/matter-test-scripts/issues/314), [318](https://github.com/project-chip/matter-test-scripts/issues/318), [272](https://github.com/project-chip/matter-test-scripts/issues/272)

Modified below test scripts:
```
Test_TC_CC_4_2
Test_TC_CC_7_2
Test_TC_PWRTL_1_1
Test_TC_SWTCH_3_2
Test_TC_OO_2_6
```

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_SWTCH_3_2.yaml [hunk 1]
```diff
@@ -724,68 +757,445 @@ tests:
...
+          [1722930820.001] [31413:31415] [TOO]   ShortRelease: {
+          [1722930820.001] [31413:31415] [TOO]     PreviousPosition: 1
+          [1722930820.001] [31413:31415] [TOO]    }
+
+
+          wait 2 seconds
+
+          echo '{"Name":"MultiPressComplete","PreviousPosition":1,"TotalNumberOfPressesCounted":2}' > /tmp/chip_all_clusters_fifo_<PID>
+
+          Verify MultiPressComplete event with PreviousPosition set to 1 AND TotalNumberOfPressesCounted is 2  on TH(all-cluster-app) log:
```
Comment (tcarmelveilleux) [inline src/app/tests/suites/certification/Test_TC_SWTCH_3_2.yaml:1025]: This whole test has so many manual steps, it is unverifiable.

**Review by tcarmelveilleux — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_OO_2_6.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                value: nodeId
+
+    - label: "Step 1: TH reads the FeatureMap attribute from DUT"
+      PICS: OO.S.F02
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x2]
```
Comment (Ashwinigrl) [inline src/app/tests/suites/certification/Test_TC_OO_2_6.yaml:42]: hasMasksSet value should be [0x4] instaed of [0x2].

File: src/app/tests/suites/certification/Test_TC_SWTCH_3_2.yaml [hunk 3]
```diff
@@ -28,6 +28,17 @@ tests:
...
+          Once TH reach the  commissionable state pls send below mentioned command on DUT(chip-tool) in RPI platform. Pls use equivalent command on the respective platform
+
+          ./chip-tool pairing onnetwork 1 20202021 --trace_decode 1
+
+          Verify the commissioning completed with success on DUT(Chip-tool)
+
+          [1650455358.501816][4366:4371] CHIP:TOO: Device commissioning completed with success
       disabled: true
 
     - label: "Note"
```
Comment (Ashwinigrl) [inline src/app/tests/suites/certification/Test_TC_SWTCH_3_2.yaml:44]: Test step 0h has to be added as per the Test plan.

File: src/app/tests/suites/certification/Test_TC_OO_2_6.yaml (hunk 2 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_OO_2_6.yaml:42]: Updated

File: src/app/tests/suites/certification/Test_TC_SWTCH_3_2.yaml (hunk 3 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_SWTCH_3_2.yaml:44]: Updated

**Review by Ashwinigrl — APPROVED** [review]

**Review by raju-apple — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_SWTCH_3_2.yaml (hunk 1 above)
Comment (cjandhyala) [reply src/app/tests/suites/certification/Test_TC_SWTCH_3_2.yaml:1033]: this is a client test case, most of the client test cases are manual for now.

**Pull request merged**